run.py

```
@@ -0,0 +1,813 @@
+import os
+import sys
+import time
+import logging
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
```

Ruff (0.8.2)

`selenium.webdriver.common.desired_capabilities.DesiredCapabilities` imported but unused

Remove unused import: `selenium.webdriver.common.desired_capabilities.DesiredCapabilities`

(F401)